src/components/Bookmark.js

```diff
@@ -0,0 +1,33 @@
+import { useState } from 'react'
+import styled from 'styled-components/macro'
+
+function Bookmark() {
+  const [bookmarkNotActiv, setBookmarkActiv] = useState(false)
+
+  return (
+    <StyledButton
+      onClick={() => {
+        setBookmarkActiv(!bookmarkNotActiv)
+      }}
+    >
+      {!bookmarkNotActiv ? (
+        <img src="/images/bookmark.png" alt="bookmark" />
+      ) : (
+        <img src="/images/bookmark-active.png" alt="active bookmark" />
```

Comment:
If the alt tag is supposed to be only a description of the icon you used, then I dont think you need an alt value at all but instead just leave it blank as it provides no value to the user:
```
<img src="/images/bookmark.png" alt="" />
) : (
<img src="/images/bookmark-active.png" alt="" />
```

If you want the alt tag to be a description of whether the user has bookmarked the object I'd change the value to something like this:
```
<img src="/images/bookmark.png" alt="" />
) : (
<img src="/images/bookmark-active.png" alt="bookmarked" />
```

Reply:
I kindly disagree. Since this is a `button` we need to add an `aria-label` to the `button` or an `alt` text to the images which are within the button. We need to tell user, what the `button` is for. How about this:
```suggestion
        <img src="/images/bookmark.png" alt="Bookmark clothing" />
      ) : (
        <img src="/images/bookmark-active.png" alt="Remove bookmark" />
```